packages/eslint-config/yaml.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  overrides: [
-    {
-      files: ['**/*.yaml', '**/*.yml'],
-      parser: 'yaml-eslint-parser',
-      plugins: ['yml'],
-      extends: ['plugin:yml/recommended'],
-    },
-  ],
-};
```

Comment:
@lucferbux why are you removing this entirely? The plan is to add yaml support in future.

Reply:
Ok, perfect, let me revert it back.